hw/ip/otbn/code-snippets/README.md

```diff
@@ -0,0 +1,8 @@
+# OTBN code snippet collection
+
+This directory contains some code snippets that give examples of how
+to do various tasks in OTBN code.
+
+  - `otbn_bn_mul64.S`: An example of how to use the 64-bit
+    multiplier (the `bn.mulqacc.*` instructions) to do 256-bit
+    multiplications.
```

Comment:
Actually this is more than multiplication. We also did the replacement of the 128x128 multiplication by our proposed sequence with the MULQACC instruction in this file, but the full functionality is  modular exponentiation and modular multiplication (both up to 2048 bit). I would suggest to:

- Change the filename to `otbn_modexp.S` or just `modexp.S` (not sure if we want a prefix)
- Add a separate example for the multiplication only. I have one [here](https://github.com/felixmiller/ot-dsim/blob/new-isa/asm/otbn_mulqacc_256x256.asm) and would prepare a PR once this one is through or you could already add it here.

Reply:
Renamed, thanks.